index.html

```diff
@@ -0,0 +1,235 @@
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-code"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-envelope-open-text"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-lightbulb"></i> <san></san>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-cash-register"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-user-tie"></i> spanDesign</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+            </div>
+
+        </div>
+    </section>
+
+
+    <section class="section-three-section">
+
+        <div class="section-three">
+            <div class="section-three-head">
+                <h1>Portfolio</h1>
+                <p>Senior graphic designer responsible for daily graphic design content for<br> a medium-sized firm. Worked regularly on graphic, layout, and<br> production materials.</p>
+            </div>
+        </div>
+
+        <div class="container-images">
+            <div><img src="images/pexels-photo-270348.jpeg" alt="">
+            </div>
+            <div><img src="images/pexels-photo-574071.jpeg" alt=""></div>
+
+            <div><img src="images/pexels-photo-577585.jpeg" alt=""></div>
+
+            <div><img src="images/pexels-photo-943096.jpeg" alt=""></div>
+
+        </div>
+        <div class="exlpore">
+
+            <button class="Explore-buuton">
+                <h2>Explore</h2>
+            </button>
+
+        </div>
+
+
+
+
+    </section>
+
+
+
+
+
+
+    <section class="section-three-section">
+
+        <div class="section-three">
+            <div class="section-three-head">
+                <h1>Our Customers</h1>
```

Comment:
Having more than one h1 tag is not in line wth the best practice. kindly rectify